partials/mixins/panel.html

```diff
@@ -0,0 +1,5 @@
+<div id="{{toggle}}-panel" class="reveal js-hidden">
+    <div class="panel-indent{{#getErrorClass}}{{toggle}}{{/getErrorClass}}">
```

Comment:
This would render something like 'panel-indentvalidation-error', but we don't need it anyway. The grey border of panel-indent should be kept when there's a validation error on a form input. The form group will have a red border that sits inside the grey border (the default behaviour).